pkgs/data/fonts/et-book/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchFromGitHub }:
+
+fetchFromGitHub rec {
+  rev = "1.0";
+  name = "et-book-${rev}";
+  owner = "jethrokuan";
```

Comment:
I think this should point to original repository from edwardtufte https://github.com/edwardtufte/et-book/tree/gh-pages/source/4-ttf

Reply:
:+1: my nix-fu wasn't strong enough when I packaged it then, let me know if there's a better way to package it now